mikupad.html

```diff
@@ -150,6 +157,27 @@
 #an-area {
 	min-height:3.5em;
 }
+.wi-textarea {
+	min-height: 4em;
+}
+
+#advancedContextPlaceholders {
+		background: var(--color-base-30);
+		margin: 1em auto;
+		padding: .25em;
+		border-radius: 2px;
+}
+#advancedContextPlaceholders tbody th {
+		text-align: right;
+}
+#advancedContextPlaceholders tbody td {
+		text-align: center;
+}

```

Comment:
Just a nitpick, but please keep the indentation at just one tab.
```suggestion
#advancedContextPlaceholders {
	background: var(--color-base-30);
	margin: 1em auto;
	padding: .25em;
	border-radius: 2px;
}
#advancedContextPlaceholders tbody th {
	text-align: right;
}
#advancedContextPlaceholders tbody td {
	text-align: center;
}
```

Reply:
Don't know how those double tabs got in there. Sorry about that.